### PR TITLE
63 post/put response enire json

### PR DESCRIPTION
**Pull request by essal0815**

### CODE REVIEW

File: packages/RatPack.package/RPTests.class/instance/testPostObjectOnApplicationResponse.st [hunk 1]
```diff
@@ -0,0 +1,15 @@
+test - REST operations
+testPostObjectOnApplicationResponse
```
Comment (He3lixxx) [inline packages/RatPack.package/RPTests.class/instance/testPostObjectOnApplicationResponse.st]: How about testPostReturnsUpdatedObject?

File: packages/RatPack.package/RPTests.class/instance/testPutObjectOnApplication.st [hunk 2]
```diff
@@ -0,0 +1,16 @@
+test - REST operations
+testPutObjectOnApplication
+	
+	| url urlId putResult postResult testJson otherTestJson|
+	app registerModel: RPTestModel.
+	url := app class apiRoutePrefix, (RPTestModel name).
+	testJson := RPTestHelper getExampleJson.
+	otherTestJson := RPTestHelper getOtherExampleJson.
+	self assert: testJson ~= otherTestJson.
```
Comment (He3lixxx) [inline packages/RatPack.package/RPTests.class/instance/testPutObjectOnApplication.st]: I don't think we need this assertion here.

File: packages/RatPack.package/RPTests.class/instance/testPutObjectOnApplication.st [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+	urlId := url, '/1'.
+	postResult := RPTestHelper post: url on: app content: testJson.
+	putResult := RPTestHelper put: urlId on: app content: otherTestJson.
+		
+	self assert: ((postResult contents contents) ~= (putResult contents contents)).
+	self assert: ((putResult contents contents) includesSubString: otherTestJson).
```
Comment (He3lixxx) [inline packages/RatPack.package/RPTests.class/instance/testPutObjectOnApplication.st]: What functionality are you trying to test here? Asserting that the responses differ but one includes the other seems not so straight forward for me. What do you think about just getting the put result and comparing it like in the test above? We're already testing post there, so we don't need to do that again. (Edit: That's exactly what the test below does. I don't quite understand what we're testing here.)

**Review by richardebeling — COMMENTED** [review]
Looks good :) I'd change the tests a bit.

**Comment by richardebeling** [comment]
fixes #63

**Pull request merged**